app/models/organization.rb

```diff
@@ -105,17 +105,8 @@ def site_name
   # ask for help in this organization
   #
   # Warning: this method does not strictly check user's permission
-  def ask_for_help_enabled?(user = nil)
```

Comment:
I'm removing this default as it comes from `assignment.submitter` and it can't be nil.